docker_push

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push USER/REPO
```

Comment:
I don't think we need this file. We can just update the .travis.yml and say `... && docker push "$DOCKER_USERNAME"/"$DOCKER_REPO"`